### DISCUSSION TITLE
Why none of the models work on weekly frequency?

### DISCUSSION THREAD

**Discussion by madscrypter** [Nixtla/neuralforecast#805] [General]
When i work with daily data I can use NBEATS and NHITS but when I set my data with freq="W" ( my data are weekly in Year-month-day format ) all the predictions turn to be NaN

**Comment by AzulGarza** [comment]
hi @madscrypter. Thanks for using `neuralforecast`. Could you paste a head of your `ds` column? Usually, weekly frequency has a particular way of encoding it depending on the day of the week. For example, if the week is represented each Thursday, the correct frequency would be `W-THU`, or `W-MON` if it is represented each Monday. By default, `W` is `W-SUN`. See [this reference](https://pandas.pydata.org/pandas-docs/stable/user_guide/timeseries.html#offset-aliases).

**Reply by madscrypter** [reply]
Thank you so much for your reply and your explaination, this is the first 20 rows of the ds:
0    2017-01-02
1    2017-01-09
2    2017-01-16
3    2017-01-23
4    2017-01-30
5    2017-02-06
6    2017-02-13
7    2017-02-20
8    2017-02-27
9    2017-03-06
10   2017-03-13
11   2017-03-20
12   2017-03-27
13   2017-04-03
14   2017-04-10
15   2017-04-17
16   2017-04-24
17   2017-05-01
18   2017-05-08
19   2017-05-15

**Reply by madscrypter** [reply]
Thank you for your tip i switched the freq='W-MON', and it works, thank you so much
